src/main/js/helpers/label.js

```diff
@@ -120,18 +121,45 @@ const renderPopup = (axisType) => {
  * @param {Array} dataTarget - Data points
  * @returns {Selection} d3 path for label shape container
  */
-const loadLabelShape = (shapeContainerPath, dataTarget) =>
-    shapeContainerPath.append(() =>
-        new Shape(dataTarget.shape || SHAPES.CIRCLE).getShapeElement(
-            getDefaultSVGProps({
-                svgStyles: `fill: ${getColorForTarget(dataTarget)};`,
-                transformFn: (scale) => `scale(${scale})`,
-                a11yAttributes: {
-                    "aria-describedby": dataTarget.key
-                }
-            })
-        )
-    );
+const loadLabelShape = (shapeContainerPath, dataTarget) => {
+    if (dataTarget.values.length === 0) {
+    } else {
+        if (dataTarget.showShapes || dataTarget.showShapes === undefined) {
```

Comment:
Why we are checking both?

1. When showshapes is defined
2. When showshapes is not defined

- What is the default value of showShapes when the consumer doesn't provide the `showShapes` property

Reply:
we do not have showShapes property defined for all graphs. So for the graphs where there is undefined showShape option, default code will get executed.